backend/src/v5/schemas/tickets/index.js

```diff
@@ -269,7 +269,7 @@ const generateCastObject = ({ properties, modules }, stripDeprecated) => {
 			if (stripDeprecated && deprecated) {
 				res[name] = Yup.mixed().strip();
 			} else if (type === propTypes.DATE || type === propTypes.PAST_DATE) {
-				res[name] = Yup.number().transform((_, val) => val.getTime());
+				res[name] = Yup.number().transform((_, val) => (val === null ? val : val?.getTime())).nullable();
```

Comment:
theoretically we cannot end up with a null date in the database isnt it ?

Reply:
yes, but this is also called by eventManager to serialise an update json for the chat server, so we will get situations where this comes in as null